sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -111,6 +110,9 @@ class QueryExecution(
   protected def stringOrError[A](f: => A): String =
     try f.toString catch { case e: AnalysisException => e.toString }
 
+  private val dateFormatter = DateFormatter()
```

Comment:
We should probably get rid of the `toHiveStructString` method for 3.0. It does not make much sense to keep it in there.

Reply:
Should I create a separate JIRA for that?

Reply:
Opened JIRA ticket: https://issues.apache.org/jira/browse/SPARK-26502